ironfish-cli/src/commands/deposit.ts

```diff
@@ -119,10 +119,14 @@ export default class Bank extends IronfishCommand {
     }
 
     const balanceResp = await this.client.getAccountBalance({ account: accountName })
-    const confirmedBalance = BigInt(balanceResp.content.confirmed)
-    const requiredBalance = CurrencyUtils.decodeIron(IRON_TO_SEND) + BigInt(fee)
+    const confirmedBalance = CurrencyUtils.decode(balanceResp.content.confirmed)
+    const requiredBalance = CurrencyUtils.decodeIron(IRON_TO_SEND) + fee
     if (confirmedBalance < requiredBalance) {
-      this.log(`Insufficient balance: ${confirmedBalance}. Required: ${requiredBalance}`)
+      this.log(
+        `Insufficient balance: ${CurrencyUtils.encode(
```

Comment:
I think were thinking about a way to make this more clear, but you should use `CurrencyUtils.renderIron()` when displaying to humans.